gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixAssignedParticipantCheck.java

```diff
@@ -52,6 +53,8 @@
 @Slf4j
 @Alias (value = "HelixParticipantCheck")
 public class HelixAssignedParticipantCheck implements CommitStep {
+  @Getter
+  @VisibleForTesting
```

Comment:
Is this variable visible ? ( private static

Reply:
The variable is accessible via Getter in the Test class. The annotation is to indicate that the Getter is added for access inside the test class.